carma/launch/guidance.launch.py

```diff
@@ -160,7 +161,7 @@ def generate_launch_description():
             ComposableNode(
                 package='route',
                 plugin='route::Route',
-                name='route_node',
+                name='route',
```

Comment:
I'm a bit skeptical of this. The route node was working on develop a week or two ago. Are you sure you tested with this change?

Reply:
this change was not required, so removed